packages/customize-widgets/src/components/sidebar-block-editor/use-sidebar-block-editor.js

```diff
@@ -25,14 +25,17 @@ function blockToWidget( block, existingWidget = null ) {
 				id: block.attributes.id,
 			};
 		} else {
+			const { encoded, hash, raw, ...rest } = block.attributes.instance;
+
 			// Widget that extends WP_Widget.
 			widget = {
 				idBase: block.attributes.idBase,
 				instance: {
-					encoded_serialized_instance:
-						block.attributes.instance.encoded,
-					instance_hash_key: block.attributes.instance.hash,
-					raw_instance: block.attributes.instance.raw,
+					...existingWidget.instance,
```

Comment:
This seems to be throwing an error for new widgets.

I tried adding a SiteOrigin Button, and this line throws because `existingWidget` is `null`.

Reply:
Oh shoot, forgot about that.

Reply:
Lets make a note to add some tests for adding/updating/removing legacy widgets 👍